### PR TITLE
Ft message fe

### PR DESCRIPTION
**Pull request by darothub**
### What this PR does (required):
- Add chat screen
- See list of available chat
- Open active chat
- See UI to engage chat
- Scrollable messages

### Screenshots / Videos (required):
Screenshot 2021-09-15 at 3 41 00 PM

### Any information needed to test this feature (required):
- cd client
- run `npm run test`

### CODE REVIEW

**Review by sundayezeilo — COMMENTED** [review]
Many files changed though as a result of your Linters enabled. This makes it take longer to review. I'd suggest you focus on the actual tickets first. You can make a separate PR for code formatting, maybe after all your tickets. :+1:

File: client/src/components/Chat/ChatContent.tsx [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    <Box className={classes.messageSideBar}>
+      <Box>
+        <Typography className={classes.username}>{'username'}</Typography>
+        {true ? (
+          <Typography className={classes.previewText}>{'latestMessageText'}</Typography>
+        ) : (
+          <Typography className={classes.previewTextSecondary}>{'latestMessageText'}</Typography>
+        )}
+      </Box>
+      <Box>{true ? <Typography variant="body2">{'10:22AM'}</Typography> : null}</Box>
```
[the remark this answers is not in the document]
Comment (darothub) [reply client/src/components/Chat/ChatContent.tsx:17]: ok I will do just that

File: client/src/pages/ChatScreen/SenderBubble.tsx [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+export default function SenderBubble(): JSX.Element {
+  const classes = useStyles();
+
+  return (
+    <Box className={classes.senderBubbleRoot}>
+      <Typography className={classes.date}>{'time'}</Typography>
+      <Box className={classes.bubble}>
+        <Typography className={classes.text}>{'text'}</Typography>
+      </Box>
+      {true ? <Avatar alt={'otherUser.username'} src={image} className={classes.avatar}></Avatar> : null}
```
[the remark this answers is not in the document]
Comment (darothub) [reply client/src/pages/ChatScreen/SenderBubble.tsx:13]: I had it there to use it when i do the integration

File: client/src/pages/ChatScreen/Conversation.tsx [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+import useStyles from './useStyles';
+
+export default function Conversation(): JSX.Element {
+  const classes = useStyles();
+
+  return (
+    <Box className={classes.conversation}>
+      {['Hello', 'yes o', 'hey', 'hi', 'hi', 'hi', 'hi', 'hi'].map((message, i) => {
+        // const time = moment(message.createdAt).format("h:mm");
+        return i % 2 == 0 ? <SenderBubble key={'SenderBubble' + i} /> : <ReceiverBubble key={'OtherUserBubble' + i} />;
```
[the remark this answers is not in the document]
Comment (darothub) [reply client/src/pages/ChatScreen/Conversation.tsx:13]: ok

[the remark this answers is not in the document]
Comment (darothub) [reply client/src/pages/ChatScreen/Conversation.tsx:13]: ok

**Comment by bonnieli** [comment]
@darothub all your PRs are actually within the 20-30 files changes, most of which because your linter keeps changing the files. For the best quality, please configure your linter so that it obliges to the application and repo's linting rules

**Comment by darothub** [comment]
> @darothub all your PRs are actually within the 20-30 files changes, most of which because your linter keeps changing the files. For the best quality, please configure your linter so that it obliges to the application and repo's linting rules

I will do just that. Thank you

**Pull request closed without merging**